packages/oui-textarea/src/textarea.html

```diff
@@ -0,0 +1,22 @@
+<div class="oui-textarea"
+    ng-class="{
+        'oui-textarea_disabled': $ctrl.disabled,
+        'oui-textarea_readonly': $ctrl.readonly
+    }">
+    <textarea class="oui-textarea__textarea"
```

Comment:
This classname is... something :)
I know it's not the tagname but maybe we could go with `oui-textarea__input` or `oui-textarea__body`

Reply:
Something I know. I thought about `input`, but it wasn't an input. I also thought about `body`  but it's not really a `body` ... `textarea` is a form controller... It could be `control`, I think.

Reply:
Yes, I find `oui-textarea__control` better too :)